utils/debug.c

```diff
@@ -384,6 +384,14 @@ void print_time_unit(uint64_t delta_nsec)
 	__print_time_unit(delta_nsec, false);
 }
 
+void print_timestamp_unit(uint64_t timestamp)
```

Comment:
Timestamp has no units, so it'd be `print_timestamp()`.

If you care about the name colision in cmds/replay.c, I suggest rename one in cmds/replay.c to `print_timestamp_field()` and call `print_timestamp()` internally with task->timestamp.